api/profile/profileRouter.js

```diff
@@ -292,4 +292,44 @@ router.delete('/:id', (req, res) => {
   }
 });
 
+/**
+ * @swagger
+ * /profile/{id}/children:
+ *  get:
+ *    summary: To fetch children for each profile
+ *    description : Get the list of children for a particular profile id
+ *    security:
+ *      - okta: []
+ *    tags:
+ *      - profile
+ *    parameters:
+ *      - $ref: '#/components/parameters/profileId'
+ *    responses:
+ *      404:
+ *        $ref: '#/components/responses/NotFound'
+ *      200:
+ *        description: A profile object
+ *        content:
+ *          application/json:
+ *            schema:
+ *              type: object
+ *              properties:
+ *                message:
+ *                  type: string
+ *                  description: A message about the result
+ *                  example: Profile id found.
+ *                profile:
+ *                  $ref: '#/components/schemas/Profile'
+ */
+router.get('/:id/children', async (req, res) => {
```

Comment:
It might be nice to make a comment here saying that you are acquiring the children _by the parent (profile id)._  I see you do mention it in the swagger doc above, but I had to really search to make sure that that was what you were doing with this endpoint.   It appears that you are getting a child by child id based on the endpoint so it might be nice to have the comment or be more descriptive with your params.